db/data/20190418114011_initialise_state_for_users.rb

```diff
@@ -0,0 +1,13 @@
+class InitialiseStateForUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.find_each(&:save!)
```

Comment:
It'd be more explicit to make this set `state` to `User.aasm.initial_state`. Not a dealbreaker though. :)

Reply:
Yes, agree